glib/src/param_spec.rs

```diff
@@ -501,6 +501,36 @@ macro_rules! define_builder {
                 self
             }
 
+            /// Mark the property as construct
+            pub fn construct(mut self) -> Self {
+                self.flags |= crate::ParamFlags::CONSTRUCT;
```

Comment:
I think `CONSTRUCT` implies `WRITE`, so you should probably also set that one here. Similarly, `CONSTRUCT_ONLY` implies `CONSTRUCT`.

Or we could leave it to the constructor to do a critical warning. Not sure what's nicer.

Reply:
I kept that as is, by assuming `READWRITE` is the default, I think leaving to the constructor to throw a critical for now is fine